### PR TITLE
Add keycode for dosa40rgb to toggle led power

### PR DESCRIPTION
**Pull request by daotakisaac**
## Description

Add a keycode for dosa40rgb to toggle led power

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by daotakisaac** [comment]
please check it ....

**Comment by noobmakers** [comment]
Heylooo

File: keyboards/dtisaac/dosa40rgb/dosa40rgb.c [hunk 1]
```diff
@@ -16,6 +16,26 @@
 
 #include "dosa40rgb.h"
 
+void keyboard_pre_init_user(void) {
+    setPinOutput(B5);
+    writePinLow(B5);
+}
```
Comment (drashna) [inline keyboards/dtisaac/dosa40rgb/dosa40rgb.c:22]:
```suggestion
void keyboard_pre_init_kb(void) {
    setPinOutput(B5);
    writePinLow(B5);
    keyboard_pre_init_user();
}
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by daotakisaac** [comment]
Hey! check it, please

**Comment by daotakisaac** [comment]
PLEASE..... Help me....

**Review by drashna — APPROVED** [review]

**Comment by daotakisaac** [comment]
Heylooooo, please help me, please, please ...... so long :(

File: keyboards/dtisaac/dosa40rgb/dosa40rgb.h [hunk 2]
```diff
@@ -18,6 +18,13 @@
 
 #include "quantum.h"
 
+enum keyboard_keycodes {
+    LED_EN = SAFE_RANGE, // Toggle LED
+    KEYMAP_SAFE_RANGE
+};
+#undef SAFE_RANGE
```
Comment (fauxpark) [inline keyboards/dtisaac/dosa40rgb/dosa40rgb.h:25]: Does this actually work? `SAFE_RANGE` is not a define, it's an entry in the quantum_keycodes enum.

Comment (daotakisaac) [reply keyboards/dtisaac/dosa40rgb/dosa40rgb.h:25]: I'm not sure,  "**zvecr**" requested to add SAFE_RANGE
[zvecr requested changes 26 days ago](https://github.com/qmk/qmk_firmware/pull/14876#discussion_r735182015)

**Pull request closed without merging**